pkg/spanconfig/spanconfigstore/span_store.go

```diff
@@ -39,7 +39,7 @@ var tenantCoalesceAdjacentSetting = settings.RegisterBoolSetting(
 // the same span config.
 var hostCoalesceAdjacentSetting = settings.RegisterBoolSetting(
 	settings.SystemOnly,
-	"spanconfig.host_coalesce_adjacent.enabled",
+	"spanconfig.storage_coalesce_adjacent.enabled",
```

Comment:
There's a spanconfig.Store in the code, so perhaps use `storage_cluster_coalesce_adjacent`? Also, "host" still appears in the variable name + surrounding commentary.

Reply:
Good call. Updated.